sdk/src/main/scala/com/horizen/account/state/BaseAccountStateView.scala

```diff
@@ -0,0 +1,34 @@
+package com.horizen.account.state
+
+import com.horizen.evm.StateDB
+
+import java.math.BigInteger
+import scala.util.Try
+
+trait BaseAccountStateView extends AccountStateReader {
+  val stateDb: StateDB
```

Comment:
Let's replace `val stateDB` with `getStateDbHandler(): Int` method.
Then in EvmMessageProc use it instead of `view.stateDB`

Reply:
Done